types/options.go

```diff
@@ -149,6 +149,14 @@ type StoreOptions struct {
 	AutoNsMaxSize uint32 `json:"auto_userns_max_size,omitempty"`
 }
 
+// isRootlessDriver returns true if the given storage driver is valid for containers running as non root
+func isRootlessDriver(driver string) bool {
+	validDrivers := map[string]bool{
+		"btrfs": true,
```

Comment:
Shouldn't vfs and overlayfs be also in this list?

Reply:
Yes, sorry, I was trying to figure out the build error with that change.